iroha/src/lib.rs

```diff
@@ -42,6 +42,9 @@ pub mod tx;
 pub mod world;
 pub mod wsv;
 
+// only for tests
+pub mod peers;
```

Comment:
I would suggest to take a look at [idiomatic way to share utility functions discussion](https://stackoverflow.com/questions/44539729/what-is-an-idiomatic-way-to-have-shared-utility-functions-for-integration-tests)